itchiodl/game.py

```diff
@@ -107,41 +108,39 @@ def do_download(self, d, token):
         """Download a single file, checking for existing files"""
         print(f"Downloading {d['filename']}")
 
-        filename = d["filename"] or d["display_name"] or d["id"]
+        filename = utils.clean_path(d["filename"] or d["display_name"] or d["id"])
+        filepath = Path(f"{self.destination_path}/{filename}")
```

Comment:
as `self.destination_path` is altready a path, you can use / operator

Reply:
This should be updated now